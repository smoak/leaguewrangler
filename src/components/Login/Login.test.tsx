import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import { Login } from '.';

describe('Login', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(
      <Login classes={{ main: 'main', avatar: 'avatar', form: 'form', submit: 'submit', paper: 'paper' }} />
    );
  });

  it('renders', () => {
    expect(component).toMatchSnapshot();
  });
});
