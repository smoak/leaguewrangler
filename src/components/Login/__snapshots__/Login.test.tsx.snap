// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders 1`] = `
<div
  className="main"
>
  <WithStyles(Paper)
    className="paper"
  >
    <WithStyles(Avatar)
      className="avatar"
    >
      <pure(LockOutlinedIcon) />
    </WithStyles(Avatar)>
    <WithStyles(Typography)
      component="h1"
      variant="h5"
    >
      Sign in
    </WithStyles(Typography)>
    <form
      className="form"
    >
      <WithStyles(FormControl)
        fullWidth={true}
        margin="normal"
        required={true}
      >
        <WithStyles(InputLabel)
          htmlFor="username"
        >
          Username
        </WithStyles(InputLabel)>
        <WithStyles(Input)
          autoComplete="username"
          autoFocus={true}
          id="username"
          name="username"
        />
      </WithStyles(FormControl)>
      <WithStyles(FormControl)
        fullWidth={true}
        margin="normal"
        required={true}
      >
        <WithStyles(InputLabel)
          htmlFor="password"
        >
          Password
        </WithStyles(InputLabel)>
        <WithStyles(Input)
          autoComplete="current-password"
          id="password"
          name="password"
          type="password"
        />
      </WithStyles(FormControl)>
      <WithStyles(FormControlLabel)
        control={
          <WithStyles(Checkbox)
            color="primary"
            value="remember"
          />
        }
        label="Remember me"
      />
      <WithStyles(Button)
        className="submit"
        color="primary"
        fullWidth={true}
        type="submit"
        variant="contained"
      >
        Sign in
      </WithStyles(Button)>
    </form>
  </WithStyles(Paper)>
</div>
`;
